From ad0fca8dc3d3e833c6c2a96c014e897ead88d8cb Mon Sep 17 00:00:00 2001
From: Roy Pledge <Roy.Pledge@freescale.com>
Date: Fri, 12 Feb 2016 11:47:03 -0500
Subject: [PATCH 1012/1429] fsl_qbman: Enable DPAA1 QBMan for ARM64 platforms

Enable the QBMan device for ARM64 platofrms.  This is needed for
ARM based SoCs that have DPAA1 such as the LS1043A

Signed-off-by: Roy Pledge <Roy.Pledge@freescale.com>
[Original patch taken from QorIQ-SDK-V2.0-20160527-yocto]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/arm64/Kconfig |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/arch/arm64/Kconfig b/arch/arm64/Kconfig
index 9db7910..7e54f04 100644
--- a/arch/arm64/Kconfig
+++ b/arch/arm64/Kconfig
@@ -271,6 +271,11 @@ endmenu
 
 menu "Bus support"
 
+config HAS_FSL_QBMAN
+        bool "Datapath Acceleration Queue and Buffer management"
+        help
+          Datapath Acceleration Queue and Buffer management
+
 config PCI
 	bool "PCI support"
 	help
-- 
1.7.5.4

